Sources/ScipioKitCore/ResolvedModels.swift

```diff
@@ -127,66 +133,92 @@ struct ResolvedModule: Hashable, Sendable {
         }
     }
 
-    var underlying: PackageManifestKit.Target
-    var dependencies: [Dependency]
-    var localPackageURL: URL
-    var packageID: PackageID
-    var resolvedModuleType: ResolvedModuleType
+    public var underlying: PackageManifestKit.Target
+    public var dependencies: [Dependency]
+    public var localPackageURL: URL
+    public var packageID: PackageID
+    public var resolvedModuleType: ResolvedModuleType
+
+    public init(
+        underlying: PackageManifestKit.Target,
+        dependencies: [Dependency],
+        localPackageURL: URL,
+        packageID: PackageID,
+        resolvedModuleType: ResolvedModuleType
+    ) {
+        self.underlying = underlying
+        self.dependencies = dependencies
+        self.localPackageURL = localPackageURL
+        self.packageID = packageID
+        self.resolvedModuleType = resolvedModuleType
+    }
 
-    var name: String {
+    public var name: String {
         underlying.name
     }
 
-    var c99name: String {
+    public var c99name: String {
         name.spm_mangledToC99ExtendedIdentifier()
     }
 
-    var xcFrameworkName: String {
+    public var xcFrameworkName: String {
         "\(c99name.packageNamed()).xcframework"
     }
 
-    var modulemapName: String {
+    public var modulemapName: String {
         "\(c99name.packageNamed()).modulemap"
     }
 
-    func recursiveModuleDependencies() throws -> [ResolvedModule] {
+    public func recursiveModuleDependencies() throws -> [ResolvedModule] {
         try topologicalSort(self.dependencies) { $0.dependencies }.compactMap { $0.module }
     }
 
-    func recursiveDependencies() throws -> [Dependency] {
+    public func recursiveDependencies() throws -> [Dependency] {
         try topologicalSort(self.dependencies) { $0.dependencies }
     }
```

Comment:
Similarly, if these functions and properties are defined as extensions in ScipioKit, we won’t need to bring any files other than ResolvedModels.swift into ScipioKitCore.